src/st/wrp/D020.c

```diff
@@ -475,7 +475,112 @@ void EntityEquipItemDrop(Entity* self) {
     }
 }
 
-INCLUDE_ASM("asm/us/st/wrp/nonmatchings/D020", func_8018E01C);
+char* BlitChar(char* str, u16* xOffset, u8* pix, u16 stride) {
+    const u16 MINSCODE = 0x8140;
+    const int FontWidth = 12;
+    const int FontHeight = 16;
+    const int FontStride = FontWidth / 2;
+
+    u16 ch;
+    s32 var_s3;
+    s32 i, j;
+    s32 letterWidth;
+    u8* chPix;
+    u8* ptr;
+    u8* dst;
+
+    ch = *str++;
+    var_s3 = 0;
+
+    if (ch >= 'a' && ch <= 'z') {
+        ch += 0x8220;
```

Comment:
This function is sort of doing the opposite of this it seems.
https://github.com/wine-mirror/wine/blob/b4280a486f7308fd1da260cd5b48b7d9e3c747c5/dlls/msvcrt/mbcs.c#L534

Shift ascii char into shift-jis lowercase fullwidth range